### ISSUE TITLE
GraphViz 3 is broken on the current released version

### ISSUE THREAD

**Issue by lf-** [schemaspy/schemaspy#944]
Please make a new release (#837), as the current release will reject graphviz versions that parse as "0.0 ".

On master, the broken version check is patched out, but the root cause is still present.

Root cause: the regex being used to parse the version can't deal with 3+ components: https://github.com/schemaspy/schemaspy/blob/6ac51a94191cbf1c20a3ffdc6f3f4f0abaeabd67/src/main/java/org/schemaspy/output/diagram/graphviz/GraphvizDot.java#L78

and it parses version 3.0.0 as "0.0 ". This is then compared to 2.something, "badGraphvizVersion", and is considered lower.

## Expected Behavior

Schemaspy doesn't reject the graphviz installation.

## Current Behavior

```
org.schemaspy.output.OutputException: Failed to produce diagram for: schemaspy/diagrams/summary/relationships.implied.
large.dot
        at org.schemaspy.output.html.mustache.diagrams.MustacheSummaryDiagramFactory.generateImpliedLarge(MustacheSumm
aryDiagramFactory.java:153)
        at org.schemaspy.output.html.mustache.diagrams.MustacheSummaryDiagramFactory.generateSummaryDiagrams(MustacheS
ummaryDiagramFactory.java:108)
        at org.schemaspy.SchemaAnalyzer.generateHtmlDoc(SchemaAnalyzer.java:351)
        at org.schemaspy.SchemaAnalyzer.analyze(SchemaAnalyzer.java:260)
        at org.schemaspy.SchemaAnalyzer.analyze(SchemaAnalyzer.java:123)
        at org.schemaspy.cli.SchemaSpyRunner.runAnalyzer(SchemaSpyRunner.java:98)
        at org.schemaspy.cli.SchemaSpyRunner.run(SchemaSpyRunner.java:87)
        at org.schemaspy.Main.main(Main.java:55)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
        at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51)
Caused by: org.schemaspy.output.diagram.DiagramException: Failed to generate summary diagram
        at org.schemaspy.output.diagram.DiagramFactory.generateSummaryDiagram(DiagramFactory.java:81)
        at org.schemaspy.output.html.mustache.diagrams.MustacheDiagramFactory.generateSummaryDiagram(MustacheDiagramFa
ctory.java:49)
        at org.schemaspy.output.html.mustache.diagrams.MustacheSummaryDiagramFactory.generateImpliedLarge(MustacheSumm
aryDiagramFactory.java:147)
        ... 15 common frames omitted
Caused by: org.schemaspy.output.diagram.DiagramException: Dot missing or invalid version
        at org.schemaspy.output.diagram.graphviz.GraphvizDot.generateDiagram(GraphvizDot.java:193)
        at org.schemaspy.output.diagram.DiagramFactory.generateSummaryDiagram(DiagramFactory.java:78)
        ... 17 common frames omitted
```

## Possible Solution

Fix the regex:

`([0-9]+\.)+[0-9]+[^\.]` seems to be correct based on my testing.

## Steps to Reproduce (for bugs)
1. Run schemaspy against graphviz 3.0.0 with basically any schema that hits the graph generator
2. Diagram generation fails

## Context

* Database type (e.g. MySQL): PostgreSQL
* Database driver version (is you use a special one):
* Your DDL (sample how to reproduce the issue):
* Full command including arguments how you run SchemaSpy:

## Your Environment

* Version used: 2.9.1, but also tried on master
* Environment name and version (e.g. Chrome 39):
* Operating System and version (e.g. Windows 10): macOS aarch64
* Java version: zulu-17.34.19, jdk 17.0.3

**Comment by marcelstoer** [comment]
I guess this should be closed as it was fixed with #945. Just waiting for #837.

**Comment by marcelstoer** [comment]
Should be closed as 6.2.0 is out now.
